controllers/grid/users/stageParticipant/form/PKPStageParticipantNotifyForm.php

```diff
@@ -38,7 +37,7 @@
 
 use PKP\core\Core;
 
-abstract class PKPStageParticipantNotifyForm extends Form
+class PKPStageParticipantNotifyForm extends Form
```

Comment:
This class is initialised in several places after the form was simplified and children classes removed.